spec/features/admin/batches/index_spec.rb

```diff
@@ -15,13 +15,6 @@
     end
   end
 
-  context 'anonymous' do
-    it "can't see it" do
-      visit '/admin/batches'
-      expect(page).to have_text('Forgot your password?')
```

Comment:
why remove this one? is it still a good check that we haven't accidentally left a page open to the public.

Reply:
It didn't seem necessary anymore, but if that is the wrong assumption I can put it back.

Reply:
Ok, re added this (and will do all of the others).